widget_driver/pubspec.yaml

```diff
@@ -1,6 +1,6 @@
 name: widget_driver
 description: A Flutter presentation layer framework, which will clean up your widget code and make your widgets testable without a need for thousands of mock objects. Let's go driving!
-version: 0.0.1
+version: 0.0.2
```

Comment:
I just bumped the version here so that we can release a new version of the `widget_driver`.
So that then later the `widget_driver_test` later can use this new version